modules/test/playwright/tests/frontend-taglib/searchPaginator.spec.ts

```diff
@@ -5,54 +5,55 @@
 
 import {Locator, expect, mergeTests} from '@playwright/test';
 
-import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
-import {featureFlagsTest} from '../../../fixtures/featureFlagsTest';
-import {isolatedSiteTest} from '../../../fixtures/isolatedSiteTest';
-import {loginTest} from '../../../fixtures/loginTest';
-import getRandomString from '../../../utils/getRandomString';
-import getPageDefinition from '../../layout-content-page-editor-web/utils/getPageDefinition';
-import getWidgetDefinition from '../../layout-content-page-editor-web/utils/getWidgetDefinition';
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {taglibSamplePageTest} from './fixtures/taglibSamplePageTest';
 
 export const test = mergeTests(
 	apiHelpersTest,
 	featureFlagsTest({
 		'LPS-178052': true,
 	}),
 	isolatedSiteTest,
-	loginTest()
+	loginTest(),
+	taglibSamplePageTest
 );
 
+const tabName = 'Search Paginator';
+
 test(
 	'Search Paginator dropdown generates page links on scrolling',
 	{tag: '@LPD-37458'},
-	async ({apiHelpers, page, site}) => {
+	async ({apiHelpers, page, site, taglibSamplePage}) => {
 		let dropdownMenuHandler: Locator;
 
-		await test.step('Create a content site and the frontend taglib sample widget', async () => {
-			const widgetDefinition = getWidgetDefinition({
-				id: getRandomString(),
-				widgetName: 'com_liferay_sample_web_portlet_SamplePortlet',
+		await test.step('Create a content site and the taglib sample widget', async () => {
+			await taglibSamplePage.setupTaglibSampleWidget({
+				apiHelpers,
```

Comment:
You can use `ApiHelpers` internally, like we do in [createAndGotoSitePage](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-editor-ckeditor-web/pages/CKEditorSamplePage.ts#L23-L41)

Reply:
Done! Thanks for the suggestion